src/components/cc-env-var-create/cc-env-var-create.js

```diff
@@ -99,6 +99,8 @@ export class CcEnvVarCreate extends LitElement {
       <cc-flex-gap>
 
         <cc-input-text
+          label=${i18n('cc-env-var-create.name.placeholder')}
```

Comment:
* :warning: You used the name translation instead of the value one
* I tried to put explicit/visible labels and no placeholder, it's not that bad actually, what do others think?

Reply:
I like it with the label, but do we really need a placeholder then?